.travis.yml

```diff
@@ -22,6 +21,10 @@ matrix:
     - rvm: ruby-head
     - rvm: jruby-head
 
+before_install:
+  - gem update --system
+  - gem install bundler
```

Comment:
why is this needed now but wasn't needed before?

Reply:
I can actually pull this back out, this was needed for `2.3` to pass as the CLI runs an old bundler that hasn't registered `truffleruby` as a `platform` so it would error in the `Gemfile`